deployment/provision.sh

```diff
@@ -1,113 +1,120 @@
 sudo apt-get install -y git
 
 
-# 
+#
 # Setup Python
-# 
-# sudo apt-get install -y python-pip
-sudo apt-get install -y python-dev python-setuptools python-imaging libssl-dev libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python-tk
-sudo easy_install pip
-# Setup newer version of Python: Currently, this project uses the native version of Python on the OS
-# cd /usr/src
-# sudo wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz
-# sudo tar xzf Python-$PYTHON_VERSION.tgz
-# cd Python-$PYTHON_VERSION
-# sudo ./configure
-# sudo make install
-# export PYTHONPATH=/usr/local/lib/python2.7/dist-packages:$PYTHONPATH
-# sudo pip install --upgrade pip
-# sudo apt-get install -y python-virtualenv # Currently, not using virtualenv
-
-
-# 
+#
+sudo apt-get install -y checkinstall \
+libreadline-gplv2-dev \
+libncursesw5-dev \
+libssl-dev \
+libsqlite3-dev \
+python3-dev \
+tk-dev \
+libgdbm-dev \
+libc6-dev \
+libbz2-dev \
+openssl \
+libffi-dev \
+python3-setuptools \
+python3-pip \
+zlib1g-dev \
+python-pil \
+libtiff5-dev \
+libjpeg8-dev \
+zlib1g-dev \
+libfreetype6-dev \
+liblcms2-dev \
+libwebp-dev \
+tcl8.6-dev \
+tk8.6-dev \
+python-tk
+
+
+#
 # Install Nginx
-# 
+#
 sudo apt-get install -y nginx
 sudo cp /$PROJECT_NAME/deployment/nginx_conf /etc/nginx/sites-available/$PROJECT_NAME
 sudo ln -s /etc/nginx/sites-available/$PROJECT_NAME /etc/nginx/sites-enabled/
-sudo pip install uwsgi
+sudo pip3 install uwsgi
 
-
-# 
+#
 # Setup Database
-# 
-sudo apt-get install -y postgresql-$PG_VERSION
-sudo apt-get install -y postgresql-contrib
-sudo apt-get install -y libpq-dev # required for psycopg2
-sudo apt-get install -y python-psycopg2
+sudo apt-get install -y postgresql-contrib libpq-dev
+sudo pip3 install psycopg2
 sudo cp /$PROJECT_NAME/deployment/pg_hba.conf /etc/postgresql/$PG_VERSION/main/
 sudo service postgresql restart
 sudo createuser -U postgres -d vagrant
 sudo createdb -U vagrant $PROJECT_NAME
 
 
-# 
+#
 # Install Python packages
 #
-# --ignore-installed is included because the Vagrant box includes binaries for several of our 
-# dependencies, but in the incorrect version. The existing installations are installed in such 
+# --ignore-installed is included because the Vagrant box includes binaries for several of our
+# dependencies, but in the incorrect version. The existing installations are installed in such
 # a way (using distutils) that pip can't upgrade them, so they have to be ignored.
 #
-sudo pip install --ignore-installed -r /$PROJECT_NAME/requirements.txt
+sudo pip3 install --ignore-installed -r /$PROJECT_NAME/requirements.txt
 
 
-# 
+#
 # Install Node and NPM for React
 #
-curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
 sudo apt-get install -y nodejs
 cd /$PROJECT_NAME
 sudo npm install -g watchify
 sudo npm install
 cd ..
 
-# 
+#
 # Populate local database
```

Comment:
Last time I checked, prepopulating the local database didn't seem to work. Could you confirm this works now?

Reply:
There were some errors setting up the schema but it seemed to be working. It's definitely had roles issues since I was last developing, but I don't know if getting those fixed up is a very priority.